src/app/course/challenge/course-challenge-create-edit/course-challenge-create-edit.component.html

```diff
@@ -1,6 +1,19 @@
 <div class="tui-container tui-container_adaptive">
+    <a
+        [routerLink]="eventId ? ['../..'] : ['..']"
+        icon="tuiIconArrowLeft"
+        size="s"
+        tuiButton
+        appearance="flat"
+    >
+        Back to List of Challenges
+    </a>
     <form (ngSubmit)="onSubmit()" [formGroup]="challengeForm">
-        <h2 class="tui-form__header">{{eventId ? 'Edit ' + event?.name : 'Create New Challenge'}}</h2>
+        <div class="flex flex-wrap gap-3 mt-3">
+            <h5 class="tui-text_h5 tui-col_stretch flex-grow">
+                {{eventId ? 'Edit ' + event?.name : 'Create New Challenge'}}
+            </h5>
+        </div>
```

Comment:
```suggestion
      <h5 class="tui-text_h5 mt-3">
          {{eventId ? 'Edit ' + event?.name : 'Create New Challenge'}}
      </h5>
```

We don't need the flex box here as there is only 1 element in it.